pysite/views/api/bot/starboard.py

```diff
@@ -110,5 +115,28 @@ def delete(self, data):
                 self.table_name,
                 message_id
             )
-            return jsonify({"success": True})
-        return jsonify({"success": False})
+            return "", 200
+        return "", 400
+
+
+class StarboardDeletionView(APIView, DBMixin):
+    path = "/bot/starboard/delete"
+    name = "bot.starboard.delete"
+    table_name = "starboard_messages"
+
+    @api_key
+    @api_params(schema=DELETE_ALL_STARBOARD, validation_type=ValidationTypes.params)
```

Comment:
I think you can omit the validation altogether if you don't take any params anyways?